accounts-tool/Cargo.toml

```
@@ -0,0 +1,10 @@
+[package]
+name = "solana-accounts-tool"
+version = "1.15.0"
+description = "Solana Accounts Tool"
+authors = ["Solana Maintainers <maintainers@solana.foundation>"]
+repository = "https://github.com/solana-labs/solana"
+homepage = "https://solana.com/"
+documentation = "https://docs.rs/solana-accounts-tool"
+license = "Apache-2.0"
+edition = "2021"
```

In light of https://github.com/solana-labs/solana/pull/29509 getting pushed, could you please rebase and then use workspace inheritance for the various Cargo.toml?
```suggestion
description = "Solana Accounts Tool"
documentation = "https://docs.rs/solana-accounts-tool"
version = { workspace = true }
authors = { workspace = true }
repository = { workspace = true }
homepage = { workspace = true }
license = { workspace = true }
edition = { workspace = true }
```

---

Nevermind, that PR got reverted for some other issue